torch/csrc/itt_wrapper.h

```diff
@@ -0,0 +1,10 @@
+#ifndef PROFILER_ITT_H
```

Comment:
Why not just put these in `torch/csrc/profiler/api.h`? Also, should be `torch::profiler` namespace

Reply:
Because `itt_mark`, `itt_range_push` and `itt_range_pop` are implemented in `torch/csrc/itt_wrapper.cpp`, to align with it, declaration of them are put in `torch/csrc/itt_wrapper.h`